src/components/pages/Intents.tsx

```diff
@@ -55,6 +55,28 @@ export const Intents = () => {
   ];
 
   const exploreIntents = [
+    {
+      name: 'NEAR Intents',
+      description:
+        'NEAR Intents are a new type of transaction that allow information, requests, assets, and actions to be exchanged between AI agents, services, and end users.',
+      icon: '/images/intents/near-intents-logo.png',
+      url: 'https://app.near-intents.org/',
+      target: '_blank',
+    },
+    {
+      name: 'Solswap',
+      description: 'Solana DEX powered by NEAR Intents',
+      icon: '/images/intents/sol-swap-logo.png',
+      url: 'https://solswap.org/',
+      target: '_blank',
+    },
+    {
+      name: 'dogecoinswap',
+      description: 'Swap native NEAR, BTC, SOL, ETH, and more for native DOGE powered by NEAR Intents',
+      icon: '/images/intents/doge-swap-logo.png',
```

Comment:
This logos look all squishy:
<img width="1346" alt="image" src="https://github.com/user-attachments/assets/ec7ad6c0-b560-487a-a590-7d32c1c0d5dc" />